internal/rbd/group/util.go

```diff
@@ -145,6 +146,12 @@ func (cvg *commonVolumeGroup) getVolumeGroupAttributes(ctx context.Context) (*jo
 		attrs = &journal.VolumeGroupAttributes{}
 	}
 
+	if attrs.GroupName == "" || attrs.CreationTime == nil {
+		log.ErrorLog(ctx, "volume group with id %v not found", cvg.id)
+
+		return nil, librbd.ErrNotFound
```

Comment:
Don't return a `librbd` error, only use `librbd` errors for checking. Return the new `ErrRBDGroupNotFound` instead. `mgr.GetVolumeGroupByID()` checks for `group.ErrRBDGroupNotFound`, which is more specific than `librbd.ErrNotFound` and therefor will not match it.

Reply:
Ack, updated the code.